--- /etc/jail.conf.d/bsd.conf	2025-12-24 21:28:33.880143000 -0600
+++ /etc/jail.conf.d/bhyve.conf	2026-05-14 09:54:13.980454000 -0500
@@ -1,5 +1,5 @@
-bsd {
-  $id = "11";
+bhyve {
+  $id = "15";

   $bridge1 = "bridge1";
   $bridge1_ip = "${bridge1}:10.0.0.${id}/24";
@@ -15,12 +15,12 @@

   vnet;

+  allow.chflags; # for pkg-upgrade
   allow.raw_sockets;
+  allow.vmm;     # for vm-init

-  mount += "/home/op $path/home/op nullfs rw 0 0";
-  mount += "/home/skhalatyan $path/home/skhalatyan nullfs rw 0 0";
   mount.devfs;
-  devfs_ruleset = 5;
+  devfs_ruleset = 10;

   enforce_statfs = 1;
