--- /etc/nsswitch.conf.sample	2025-12-05 07:03:34.161846000 -0600
+++ /etc/nsswitch.conf	2025-12-04 08:27:47.553363000 -0600
@@ -1,12 +1,12 @@
 #
 # nsswitch.conf(5) - name service switch configuration file
 #
-group: compat
+group: files ldap
 group_compat: nis
 hosts: files dns
 netgroup: compat
 networks: files
-passwd: compat
+passwd: files ldap
 passwd_compat: nis
 shells: files
 services: compat
