--- /usr/local/etc/nss_ldap.conf.sample	2025-08-10 10:17:40.000000000 -0500
+++ /usr/local/etc/nss_ldap.conf	2025-09-28 08:56:50.439652000 -0500
@@ -12,10 +12,10 @@
 # space. How long nss_ldap takes to failover depends on
 # whether your LDAP client library supports configurable
 # network or connect timeouts (see bind_timelimit).
-host 127.0.0.1
+# host 127.0.0.1
 
 # The distinguished name of the search base.
-base dc=padl,dc=com
+base dc=lab,dc=net
 
 # Another way to specify your LDAP server is to provide an
 # uri with the server name. This allows to use
@@ -24,6 +24,7 @@
 #uri ldaps://127.0.0.1/   
 #uri ldapi://%2fvar%2frun%2fldapi_sock/
 # Note: %2f encodes the '/' used as directory separator
+uri ldap://192.168.1.90
 
 # The LDAP version to use (defaults to 3
 # if supported by client library)
