--- /usr/local/etc/nss_ldap.conf.sample	2025-12-01 12:15:48.000000000 -0600
+++ /usr/local/etc/nss_ldap.conf	2025-12-03 18:00:32.458154000 -0600
@@ -12,10 +12,10 @@
 # space. How long nss_ldap takes to failover depends on
 # whether your LDAP client library supports configurable
 # network or connect timeouts (see bind_timelimit).
-host 127.0.0.1
+host ldap.lab.net

 # The distinguished name of the search base.
-base dc=padl,dc=com
+base dc=lab,dc=net

 # Another way to specify your LDAP server is to provide an
 # uri with the server name. This allows to use
