--- /usr/local/etc/nslcd.conf.sample	2026-07-07 12:24:18.000000000 +0000
+++ /usr/local/etc/nslcd.conf	2026-07-21 02:15:14.745686000 +0000
@@ -15,14 +15,14 @@
 #uri ldaps://127.0.0.1/
 #uri ldapi://%2fvar%2frun%2fldapi_sock/
 # Note: %2f encodes the '/' used as directory separator
-uri ldap://127.0.0.1/
+uri ldap://ldap.lab.net/
 
 # The LDAP version to use (defaults to 3
 # if supported by client library)
 #ldap_version 3
 
 # The distinguished name of the search base.
-base dc=example,dc=com
+base dc=lab,dc=net
 
 # The distinguished name to bind to the server with.
 # Optional: default is to bind anonymously.
@@ -59,8 +59,8 @@
 #idle_timelimit 3600
 
 # Use StartTLS without verifying the server certificate.
-#ssl start_tls
-#tls_reqcert never
+ssl start_tls
+tls_reqcert try
 
 # CA certificates for server certificate verification
 #tls_cacertdir /etc/ssl/certs
